### DISCUSSION TITLE
Debugging Node adapter - production build complains on imported packages

### DISCUSSION THREAD

**Discussion by PlkMarudny** [sveltejs/kit#11907] [Q&A] [answered]
I have a SvelteKit application that does not work after build. It is supposed to be run in a container, Node driver. At one of the server endpoints I use [bullmq ](https://docs.bullmq.io/)client, which, when hit, produces an error:

```
gui  | ReferenceError: module is not defined in ES module scope
gui  | This file is being treated as an ES module because it has a '.js' file extension and '/app/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
gui  |     at getPkgJsonDir (file:///app/build/server/chunks/_server-DJYoxZ8R.js:47284:40)
gui  |     at new ScriptLoader (file:///app/build/server/chunks/_server-DJYoxZ8R.js:46970:25)
gui  |     at file:///app/build/server/chunks/_server-DJYoxZ8R.js:47336:1
gui  |     at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
gui  |     at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
gui  |     at async respond (file:///app/build/server/index.js:3524:22)
gui  |     at async Array.ssr (file:///app/build/handler.js:1243:3)
```

During development I also use Node driver, and a preview served from `build` directory seems to be working fine.  How to debug this?

**Comment by leoj3n** [comment]
@PlkMarudny Have you tried looking at the code in the files on the lines specified? I don't think this is an issue with `bullmq`. It sounds like maybe the `package.json` is different on dev than in production, or your node driver is configured different locally than in remote.

**Comment by PlkMarudny** [comment, marked as answer]
Well, I solved it. Moving `bullmq` package from dev dependencies to dependencies was the solution.
